.github/workflows/test-abi_wasm.yml

```diff
@@ -15,17 +15,23 @@ concurrency:
 jobs:
   build:
     runs-on: ubuntu-latest
+    env:
+      CACHED_PATH: /tmp/nix-cache
 
     steps:
       - name: Checkout sources
         uses: actions/checkout@v3
 
       - name: Setup Nix
-        uses: ./.github/actions/nix
+        uses: cachix/install-nix-action@v22
         with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-          nix-cache-name: ${{ vars.NIX_CACHE_NAME }}
-          cachix-auth-token: ${{ secrets.CACHIXAUTHTOKEN }}
+          nix_path: nixpkgs=channel:nixos-23.05
+          github_access_token: ${{ secrets.GITHUB_TOKEN }}
+
+      - uses: cachix/cachix-action@v12
+        with:
+          name: barretenberg
+          authToken: "${{ secrets.CACHIX_AUTH_TOKEN }}"
```

Comment:
Looks like most of this PR is .yml changes. Was this the result of a bad merge or something?

Reply:
Yeah, this is a bad merge. It's undoing a lot of my changes from yesterday.

Reply:
@TomAFrench #3366, #3368 after these pull requests, the CI starts failing with this error:

![image](https://github.com/noir-lang/noir/assets/85340010/d3e8dccd-7f5f-4bda-94d8-f74930d23d15)

Reply:
@kobyhallx @kevaundray is `vars.NIX_CACHE_NAME` not available to forks?

Reply:
Lets replace its usage with the actual value and open an issue if we want to re-introduce this so its not blocking

Reply:
#3398